tests/suites/test_suite_pkcs7.function

```diff
@@ -121,12 +121,12 @@ void pkcs7_verify(char *pkcs7_file,
         TEST_EQUAL(res, 0);
     }
 
-    res = stat(filetobesigned, &st);
-    TEST_EQUAL(res, 0);
-
     file = fopen(filetobesigned, "rb");
     TEST_ASSERT(file != NULL);
 
+    res = fstat(fileno(file), &st);
```

Comment:
This adds a dependency on yet another system function: our code base doesn't currently use `fstat`, whereas there is a call to `stat` in the X.509 library code. Please avoid adding dependencies on system functions as they reduce portability.

A more portable and less race-prone way to obtain the file size would be to use use `fseek` and `ftell`. That's what the library `mbedtls_xxx_parse_file` functions do. There's still the possibility of a race if the file is modified (as opposed to renamed), but such is life. The consequence is a functional error, not a security hole.